packages/app/src/debug/empty/DebugEmptyView.vue

```diff
@@ -2,17 +2,23 @@
   <div class="flex flex-col mx-auto my-45px max-w-680px items-center">
     <div class="flex flex-col items-center justify-evenly">
       <div><i-cy-box-open_x48 class="icon-dark-gray-500 icon-light-indigo-100" /></div>
-      <div class="flex flex-col mx-[20%] mt-25px mb-20px items-center">
+      <div class="flex flex-col mt-25px mb-20px max-w-640px items-center">
         <div class="font-medium my-5px text-center text-gray-900 text-18px">
           {{ title }}
         </div>
         <div class="font-normal my-5px text-center leading-relaxed text-16px text-gray-600">
-          {{ description }} <ExternalLink
-            v-if="helpLinkText && helpLinkHref"
-            :href="helpLink"
-          >
-            {{ helpLinkText }}
-          </ExternalLink>
+          {{ description }}
+          <span class="ml-4px">
+            <span class="sr-only">
+              {{ helpLinkSrText }}
+            </span>
+            <ExternalLink
+              v-if="helpLinkHref"
+              :href="helpLink"
+            >
+              {{ t('links.learnMoreButton') }}
+            </ExternalLink>
+          </span>
```

Comment:
```suggestion
          <span class="ml-4px">
            <ExternalLink
              v-if="helpLinkHref"
              :href="helpLink"
            >
              {{ t('links.learnMoreButton') }}
              <span class="sr-only">
                {{ helpLinkSrText }}
              </span>
            </ExternalLink>
          </span>
```

Thanks @marktnoonan for pointing out the A11y concerns!  In order for this to work the best, the text should go inside the link so that screen readers will announce it as part of the link text.   For this solution, the screen reader only part of the text should be updated to just be the rest of the sentence.  One example when rendered would look like:
```
<a href="{{url}}>Learn more <span class="sr-only">about recording a run to Cypress Cloud</span></a>
```

The video ["Write more accessible link text"](https://www.youtube.com/watch?v=_SfEUKlN-7Q) that I found does a good job explaining the problem.  Our solution is a little different than the one in the video, but it might be good pattern to recommend for future changes.

Reply:
good point! I moved the screen reader text inside of the link and updated the translations 👍🏻